package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "iconv-lite-umd",
+  "version": "0.5.1",
```

Comment:
Not sure what version scheme you are following for `-umd` packages. Decided to keep it tied to the original package version.

Reply:
Yes that is good!